script.js

```diff
@@ -1,4 +1,110 @@
+//Dice Roll function
+var diceRoll = function () {
+  return Math.ceil(Math.random() * 6);
+};
+
+//Starting game mode
+gameMode = `rollingDice`;
+
+//Player 1 roll
+var p1Roll = [diceRoll(), diceRoll()];
+//Player 2 roll
+var p2Roll = [diceRoll(), diceRoll()];
```

Comment:
Interesting choice of rolling the dice once the page loads!  

I think for the purposes of this 2 player game it works! However always prepare for multiplayer so refactoring won't be such a headache . Heh!